erc20-price/substreams.yaml

```diff
@@ -0,0 +1,27 @@
+specVersion: v0.1.0
```

Comment:
I thought you were going to make this a generic (non-substream) Rust crate?

Reply:
Not possible though, since all contract calls need to go through substreams-ethereum eth_call, otherwise it cannot run in substreams